### ISSUE TITLE
[feature-request] Implement wisdom runes XP bonus

### ISSUE THREAD

**Issue by 13k** [Yewchi/vulft#21]
Bots should have a strategy to pick wisdom runes up.

Also, (briefly tested this, not confirmed) it seems allies don't gain XP whenever the player picks up wisdom runes.

**Comment by Yewchi** [comment]
Not possible as of 7.33(a) Action_PickUpRune(botuserdata, runeID) has not been updated in Dota 2's API.

The bots perform no action.

**Comment by Yewchi** [comment]
https://github.com/Yewchi/vulft/blob/c7fe6e8f49692bfeaa448d64e84ac5445bf2d2d5/lib_task/blueprint/dawdle.lua#L231-L256

**Comment by Yewchi** [comment]
Dawdle -> ability_generics.lua. comment states status, (ignore Outpost function that works):
https://github.com/Yewchi/vulft/blob/c7fe6e8f49692bfeaa448d64e84ac5445bf2d2d5/lib_hero/ability/ability_generics.lua#L274-L378

Bots will move to only grab runes that may be or are spawned (fog of war) at the bounty and powerup locations:
https://github.com/Yewchi/vulft/blob/c7fe6e8f49692bfeaa448d64e84ac5445bf2d2d5/lib_util/debug.lua#L717-L721


Just dumping 7.33 API issues as referenced in here, runes were tested by dominating bots with

**Comment by 13k** [comment]
What about the second issue of not distributing XP when the player picks a wisdom rune?

I can test again to confirm it. Should I open a separate issue for that?

**Comment by Yewchi** [comment]
Wisdom runes don't do that, in any game mode. It's like a collectable Tome of Knowledge for one player.

**Comment by 13k** [comment]
You're right. For some reason I assumed they were like bounty runes (gold for the whole team).
